frontend/src/App.js

```diff
@@ -1,32 +1,44 @@
 import React from "react";
 import NavBar from "./components/NavBar";
-import { Switch, Route } from "react-router-dom";
+import { Switch, Route, useLocation } from "react-router-dom";
 import Home from "./pages/Home";
 import CampaignInfos from "./pages/CampaignInfos";
 import Ambassadors from "./pages/Ambassadors";
 import AmbassadorInfos from "./pages/AmbassadorInfos";
 import Causes from "./pages/Causes";
 import CauseInfos from "./pages/CauseInfos";
 import Footer from "./components/Footer";
+import HomeBO from "./pages/HomeBO";
+import CampaignsPageBO from "./pages/CampaignsPageBO";
+import AmbassadorsPageBO from "./pages/AmbassadorsPageBO";
+import CausesPageBO from "./pages/CausesPageBO";
+import CampaignPage from "./pages/CampaignPage";
+import NavBarBO from "./components/NavBarBO";
 import "./App.scss";
+
 const { siteTitle } = require("./conf");
 
 
 function App() {
   document.title = siteTitle;
+  const location = useLocation();
 
   return (
     <div className="App">
-      <NavBar />
+      {location.pathname.includes("admin") ? <NavBarBO /> : <NavBar />}
```

Comment:
#notBlocking - The idea is good, but it's a security breach: nothing forbids your user from typing "admin" in the url (`/admin` was for a very long time the default route to administration for WP websites... Wannabe hackers may try and type this at "random")
I'd at least sort my admin routes as `PrivateRoute` (see `old_roleet` for that), to limit what attackers could access through this